.cbmc-batch/source/make_common_data_structures.c

```diff
@@ -146,3 +146,9 @@ enum aws_cryptosdk_sha_version aws_cryptosdk_which_sha(enum aws_cryptosdk_alg_id
         default: return AWS_CRYPTOSDK_NOSHA;
     }
 }
+
+void ensure_cryptosdk_keyring_has_allocated_members(
+    struct aws_cryptosdk_keyring *keyring, const struct aws_cryptosdk_keyring_vt *vtable) {
+    keyring->refcount.value = can_fail_malloc(sizeof(size_t));
```

Comment:
`keyring->refcount.value = can_fail_malloc(sizeof(* keyring->refcount.value));` is self-documenting

Reply:
Ahh.

Reply:
for keyrings? yes!